### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by XedinUnknown**
- Do not ignore Prophecy. Seems that this dependency is no longer getting installed, which is causing Psalm to complain.
- Remove a deprecated attribute in Psalm config, which was causing it to complain as well.

### CODE REVIEW

**Comment by XedinUnknown** [comment]
@mecha, @Biont, you wanna try this to confirm whether it breaks your stuff? Looks like the repo went out of date while Psalm inspections improved.

File: src/Factories/StringService.php [hunk 1]
```diff
@@ -42,6 +43,32 @@ public function __construct(string $format, array $dependencies = [])
...
+     * Retrieve a service _as a string_ with the specified name from the given container.
+     *
+     * @param string $serviceName Name of the service to retrieve.
+     * @param ContainerInterface $c The container to retrieve the service from.
+     *
+     * @return string The string representation of the service.
+     *
+     * @throws UnexpectedValueException If service could be converted to string.
+     */
+    protected function resolveString(string $serviceName, ContainerInterface $c): string
```
Comment (mecha) [inline src/Factories/StringService.php:56]: I don't like that this method is duplicated across 2 classes. Perhaps we should add a utility function somewhere that both of these classes can call.

**Review by mecha — CHANGES_REQUESTED** [review]

File: src/Factories/StringService.php (hunk 1 above)
Comment (mecha) [reply src/Factories/StringService.php:56]: Looking at this again and - I'm not sure what I thought I saw - I don't see the other "occurrence" of this code. I must have made a mistake.

**Review by mecha — APPROVED** [review]
I've tested these changes on my local copy of #15 and can confirm that the changes do not add any undesired side-effects.

**Comment by Biont** [comment]
@XedinUnknown @mecha It took me a while, but I was also able to confirm that there are no apparent side-effects/breakages introduced in this branch

**Review by Biont — APPROVED** [review]

**Comment by XedinUnknown** [comment]
Thanks! I think we have all the assurance we need right now.

**Pull request merged**